app/javascript/dashboard/store/modules/conversations/index.js

```diff
@@ -145,7 +150,7 @@ export const mutations = {
   // Update assignee on action cable message
   [types.default.UPDATE_ASSIGNEE](_state, payload) {
```

Comment:
ASSIGN_AGENT is similar to UPDATE_ASSIGNEE right? Can we merge these two?

Reply:
It takes a refactoring to change it.